/**:
  ros__parameters:
    gnss_pose_timeout: 3.0 # [sec]
    stop_check_duration: 3.0 # [sec]
    ekf_enabled: true
    gnss_enabled: true
    yabloc_enabled: true
    ndt_enabled: true
    stop_check_enabled: true

    # from gnss
    gnss_particle_covariance:
      [
        1.0, 0.0, 0.0,  0.0,  0.0,  0.0,
        0.0, 1.0, 0.0,  0.0,  0.0,  0.0,
        0.0, 0.0, 0.01, 0.0,  0.0,  0.0,
        0.0, 0.0, 0.0,  0.01, 0.0,  0.0,
        0.0, 0.0, 0.0,  0.0,  0.01, 0.0,
        0.0, 0.0, 0.0,  0.0,  0.0,  10.0,
      ]

    # output
    output_pose_covariance:
      [
        1.0, 0.0, 0.0,  0.0,  0.0,  0.0,
        0.0, 1.0, 0.0,  0.0,  0.0,  0.0,
        0.0, 0.0, 0.01, 0.0,  0.0,  0.0,
        0.0, 0.0, 0.0,  0.01, 0.0,  0.0,
        0.0, 0.0, 0.0,  0.0,  0.01, 0.0,
        0.0, 0.0, 0.0,  0.0,  0.0,  0.2,
      ]
